openpype/pipeline/publish/lib.py

```diff
@@ -623,12 +638,34 @@ def get_instance_staging_dir(instance):
     Returns:
         str: Path to staging dir of instance.
     """
+    staging_dir = instance.data.get('stagingDir')
+    if staging_dir:
+        return staging_dir
+
+    anatomy_data = instance.data.get("anatomy_data")
+    anatomy = instance.data.get("anatomy")
```

Comment:
I think anatomy is only on context
```suggestion
    anatomy = instance.context.data.get("anatomy")
```